deploy/helm/sumologic/README.md

```diff
@@ -248,7 +249,7 @@ The following table lists the configurable parameters of the Sumo Logic chart an
+| `otelcol.monitoring.metrics.enabled`                                      | Enable or disable generation of the metrics from Collector.                                                                                                                                                                                                                                                                                                                                                                                   | `false`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
```

Comment:
In values.yaml `otelcol.monitoring.metrics.enabled` is set to true, see [here](https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/1702/files#diff-da997ab241c9344f2bb085822fe04e4e43eaf0f8034a9d15dc3942b8a5fbe046R2496)

Reply:
good catch :+1: